### ISSUE TITLE
`pip install scikit-bio` fails in new environment without numpy

### ISSUE THREAD

**Issue by lrvdijk** [biocore/scikit-bio#1790]
When I create a completely new Conda environment (python=3.9), and then try to install scikit-bio through `pip` I receive the following error:

```
$ pip install scikit-bio
Collecting scikit-bio
  Using cached scikit-bio-0.5.6.tar.gz (8.4 MB)
    ERROR: Command errored out with exit status 1:
     command: /opt/homebrew/Caskroom/miniconda/base/envs/strainge/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/_z/4w3qtphj3kj949vtrhwrk4wm0000gn/T/pip-install-ej8npkt6/scikit-bio_e503412b1de440d1b80bc0a890ffb2a2/setup.py'"'"'; __file__='"'"'/private/var/folders/_z/4w3qtphj3kj949vtrhwrk4wm0000gn/T/pip-install-ej8npkt6/scikit-bio_e503412b1de440d1b80bc0a890ffb2a2/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/_z/4w3qtphj3kj949vtrhwrk4wm0000gn/T/pip-pip-egg-info-lpnck2g9
         cwd: /private/var/folders/_z/4w3qtphj3kj949vtrhwrk4wm0000gn/T/pip-install-ej8npkt6/scikit-bio_e503412b1de440d1b80bc0a890ffb2a2/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/_z/4w3qtphj3kj949vtrhwrk4wm0000gn/T/pip-install-ej8npkt6/scikit-bio_e503412b1de440d1b80bc0a890ffb2a2/setup.py", line 20, in <module>
        import numpy as np
    ModuleNotFoundError: No module named 'numpy'

```

Why does pip not automatically install this dependency?

I know I can install scikit-bio through conda too, but I thought this was still curious. I'm currently writing a tool that depends on scikit-bio, and which doesn't have a conda package yet, so when I try to `pip install my_tool` with scikit-bio as dependency, it fails.

**Comment by wasade** [comment]
Numpy headers are used during compilation, which make it necessary to have to satisfy `setup.py` execution. We'd welcome a PR to resolve this issue if you're able?

**Comment by wasade** [comment]
This is solved, see #1671, and should be addressed in the pending release

**Comment by marsfro** [comment]
I was trying a lot, but It helped just restart computer ))
